src/utils/response.ts

```diff
@@ -0,0 +1,32 @@
+import { getTrackerMode, TrackerMode } from "./env_helper";
+
+export enum TrackerResponseType {
+  Success = "success",
+  Failed = "failed",
+}
+
+export class TrackerResponse {
+  status: number = 200;
+  state: TrackerResponseType = TrackerResponseType.Success;
+  mode: TrackerMode = getTrackerMode();
+  message: string = "";
+  data: any = {};
```

Comment:
what `data` could be returned in a TrackerResponse?

Reply:
not sure yet. the snowplow-tracker only returns an object that has add, addDict, addJson, and build function, so I don't really know what we can return. i'll take it out until we have a better use case, right?

Reply:
sounds good - can leave it out for now

Reply:
actually i'm using data to return the payload properties when in test mode. lets keep it for now and we can figure out what we can return for a prod data response

Reply:
sorry, i think you're right. it's too redundant to have data, i'll remove it